### PR TITLE
Instructions for connecting to VPN from Gnome-running Linuxes

### PR DESCRIPTION
**Pull request by gigawhitlocks**
#### Summary

This change adds some instructions for those running Linux + Gnome to connect to the new Cloud VPN.

### CODE REVIEW

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 1]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
 
 7. After following these steps you should be able to connect to VPN and then to resolve private DNS entries.
 
+## Gnome VPN Client
+1. Go to the [VPN server](https://pritunl.core.cloud.mattermost.com) and select **Sign in with OneLogin**
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:80]:
```suggestion
1. Go to the [VPN server](https://pritunl.core.cloud.mattermost.com) and select **Sign in with OneLogin**.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 2]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
 7. After following these steps you should be able to connect to VPN and then to resolve private DNS entries.
 
+## Gnome VPN Client
+1. Go to the [VPN server](https://pritunl.core.cloud.mattermost.com) and select **Sign in with OneLogin** 
+
+2. Connect with your OneLogin username and password and when prompted input the OTP (One Time Password).
+
+3. Click `Download Profiles` and save the `.tar` file to your filesystem.
+
+4. Extract the `.tar` file ( `tar xf yourusername.tar`) and note the location of the `.ovpn` file within.
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:86]:
```suggestion
4. Extract the `.tar` file (`tar xf yourusername.tar`) and note the location of the `.ovpn` file.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 3]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+5. Open the Network section in the Gnome Settings manager and there will be a VPN section. Click the `+` to create a new VPN connection.
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:88]:
```suggestion
5. Open the Gnome Settings manager and navigate to **Network > VPN**. Click the `+` to create a new VPN connection.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 4]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+6. Choose `Import from file...`
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:90]:
```suggestion
6. Choose **Import from file...**.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 5]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+7. Select the `.ovpn` file downloaded earlier through the file picker.
+
+8. Open the IPv4 tab and select `Use this connection only for resources on its network`
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:94]:
```suggestion
8. Open the IPv4 tab and select **Use this connection only for resources on its network**.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 6]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+9. Open the IPv6 tab and select `Use this connection only for resources on its network`
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:96]:
```suggestion
9. Open the **IPv6** tab and select **Use this connection only for resources on its network**.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 7]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+10. If desired, rename the VPN to something friendlier in the `Name` field under the `Identity` tab.
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:98]:
```suggestion
10. If desired, rename the VPN to something friendlier in **Identity > Name**.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 8]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+11. Save the configuration by pressing `Add` at the top right of the window.
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:100]:
```suggestion
11. Choose **Add** to save the configuration. 
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 9]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
...
+
+12. From now on, enable the VPN through the taskbar picker in the upper right of Gnome.
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:102]:
```suggestion
12. From now on, enable the VPN through the taskbar picker in the upper right corner of Gnome.
```

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 10]
```diff
@@ -76,6 +76,30 @@ Then connect with your OneLogin username and password and when prompt put the OT
 
 7. After following these steps you should be able to connect to VPN and then to resolve private DNS entries.
```
Comment (justinegeffen) [inline site/content/internal/infrastructure/vpn-cloud.md:77]:
```suggestion
7. After following these steps you should be able to connect to VPN and resolve private DNS entries.
```

**Comment by gigawhitlocks** [comment]
Thank you for the review @justinegeffen I have accepted your suggestions. Let me know if there's anything else you need from me so we can get this merged :)

**Comment by justinegeffen** [comment]
Thanks, @gigawhitlocks, I think it's good to go. :)

**Pull request merged**